tests/test_reqs_hints.py

```diff
@@ -0,0 +1,56 @@
+"""Test for Requirements and Hints in cwltool."""
+import json
+from io import StringIO
+
+from cwltool.main import main
+from .util import get_data
+
+
+def test_workflow_reqs_are_evaluated_eagerly_default_args() -> None:
+    """Test that a Workflow process will evaluate the requirements eagerly.
+    Uses the default input values.
```

Comment:
Was this accepted by `make diff_pydocstyle_report`?

Reply:
Ah, it was not. Fixed. Thanks @mr-c !